ecommerce/templates/oscar/basket/basket.html

```diff
@@ -6,23 +6,13 @@
 {% load static %}
 
 {% block title %}
-{% trans 'Basket' as tmsg %}{{ tmsg | force_escape }}
+{% trans 'Basket' %}
 {% endblock title %}
 
 {% block navbar %}
     {% include 'edx/partials/_student_navbar.html' %}
 {% endblock navbar %}
 
-{% block pre_app_js %}
```

Comment:
Why is this being removed?

Reply:
Exactly Moses, this was what was agreed with Felipe after having reviewed the changes that were made from the Hawtorn version to Juniper. The best option was to go back to how it was in the template in the Hawtorn version

Reply:
![Screenshot from 2021-06-29 16-37-33](https://user-images.githubusercontent.com/66017940/123870701-584f1980-d8f8-11eb-84bc-9a9d7ba18732.png)

Reply:
And what about the js part? this was added on a different commit [5cb2818](https://github.com/eduNEXT/edunext-ecommerce/commit/5cb2818850a12fbf704e240b9f7b10178a5221b0#diff-9c1c0bad70d4bfd2440739de8f85d617ef33d9963c65c8fc7bd95964775897f8)? Do you really need to remove it?

Reply:
You're right, It doesn't need to be remove , I just fixed
Thanks!!!